src/main/java/org/jabref/gui/GUIGlobals.java

```diff
@@ -48,13 +48,12 @@ public static void init() {
         GUIGlobals.updateEntryEditorColors();
 
         IconTheme.loadFonts();
-        GUIGlobals.currentFont = new Font(Globals.prefs.get(JabRefPreferences.FONT_FAMILY),
-                                          Globals.prefs.getInt(JabRefPreferences.FONT_STYLE), Globals.prefs.getInt(JabRefPreferences.FONT_SIZE));
+        GUIGlobals.currentFont = new Font(Globals.prefs.get(JabRefPreferences.FONT_FAMILY), Globals.prefs.getInt(JabRefPreferences.FONT_SIZE));
     }
 
-    public static void setFont(int size) {
-        currentFont = new Font(currentFont.getFamily(), currentFont.getStyle(), size);
+    public static void setFont(double size) {
+        currentFont = new Font(currentFont.getFamily(), size);
         // update preferences
-        Globals.prefs.putInt(JabRefPreferences.FONT_SIZE, size);
+        Globals.prefs.putDouble(JabRefPreferences.FONT_SIZE, size);
```

Comment:
I guess with this we will run into backwards compatibility issues similar to the ones we had with a different preference option.

Reply:
Hmm, yeah will look if it makes sense to either convert to int or store in new settings

Reply:
Naive question: can't be stick to Int and just convert to Double during setting the stuff? I would assume that 14.5pt is currently not used --> future work ^^

Reply:
I changed it already to int again.         Globals.prefs.putInt(JabRefPreferences.FONT_SIZE, size);
So it's fine